scripts/simplify_network.py

```diff
@@ -269,6 +269,7 @@ def simplify_links(n, costs, config, output, aggregation_strategies=dict()):
     logger.info("Simplifying connected link components")
 
     if n.links.empty:
+        pd.DataFrame({}).to_csv(output.connection_costs)
```

Comment:
Rather than using the to_csv on an empty dataframe, it is maybe more error-resilient:
```with open(path, "w") as fp:
       pass
```
I agree no release_note is needed.

Nice :)

Reply:
> Rather than using the to_csv on an empty dataframe, it is maybe more error-resilient:
> 
> ```python
> with open(path, "w") as fp:
>     pass
> ```
> 
> I agree no release_note is needed.
> 
> Nice :)

Agree, it looks better :) Thanks, @davide-f!
Changed